app/views/admin/configs/show.html.erb

```diff
@@ -642,7 +642,7 @@
                 <%= f.fields_for :meta_keywords do |meta_keywords_field| %>
                   <% SiteConfig.meta_keywords.each do |area, keywords| %>
                     <div>
-                      <%= meta_keywords_field.label "#{area}*" %>
```

Comment:
I wasn't sure why we had a `*` appended to these, but it sort of makes it look like they're required fields. I removed this since it could potentially be confusing and didn't seem super informative
<img width="785" alt="Screen Shot 2020-10-07 at 1 31 02 PM" src="https://user-images.githubusercontent.com/6921610/95384381-614b1700-08a1-11eb-9923-dedf3eb30317.png">